### PR TITLE
preparation IH

### PR DESCRIPTION
**Pull request by eliegaboriau**
This PR adds admin creation tasks and udpates CI/CD

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      - uses: OpenSourcePolitics/build-and-push-images-action@master
+        with:
+          registry: ${{ vars.REGISTRY_ENDPOINT }}
+          namespace: ${{ vars.REGISTRY_NAMESPACE }}
+          password: ${{ secrets.TOKEN }}
+          image_name: ${{ vars.IMAGE_NAME }}
+          tag: "develop-${{ github.sha }}"
+  generate_release:
+    name: Generate release
+    needs: [ lint, tests, test_build ]
```
Comment (armandfardeau) [inline .github/workflows/ci_cd.yml:133]:
```suggestion
    needs: [ lint, tests, system_tests, test_build ]
```

**Review by armandfardeau — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          image_name: ${{ vars.IMAGE_NAME }}
+          tag: ${{ github.ref }}
+          password: ${{ secrets.TOKEN }}
+          database_username: ${{ env.DATABASE_USERNAME }}
+          database_password: ${{ env.DATABASE_PASSWORD }}
+          database_host: ${{ env.DATABASE_HOST }}
+  build_and_push_image_dev:
+    name: Build and push image to Registry
+    if: "github.ref == 'refs/heads/develop'"
+    needs: [ lint, tests, test_build ]
```
Comment (armandfardeau) [inline .github/workflows/ci_cd.yml:121]:
```suggestion
    needs: [ lint, tests, system_tests, test_build ]
```

**Comment by armandfardeau** [comment]
Wait for https://github.com/mainio/decidim-module-term_customizer/pull/100 to be resolved

File: config/environments/production.rb [hunk 3]
```diff
@@ -129,4 +129,6 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.deface.enabled = ENV['DEFACE_ENABLED'] == 'true'
```
Comment (armandfardeau) [inline config/environments/production.rb:133]:
```suggestion
  config.deface.enabled = ENV.fetch('DEFACE_ENABLED',  "false") == 'true'
```

**Review by armandfardeau — CHANGES_REQUESTED** [review]

**Comment by armandfardeau** [comment]
@eliegaboriau Is this ready for review?

**Comment by eliegaboriau** [comment]
Hello @armandfardeau , it is now

**Review by Quentinchampenois — APPROVED** [review]

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**